google-oauth-client-jetty/pom.xml

```diff
@@ -87,8 +87,9 @@
       <artifactId>google-oauth-client-java6</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.mortbay.jetty</groupId>
-      <artifactId>jetty</artifactId>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-server</artifactId>
+      <version>8.2.0.v20160908</version>
```

Comment:
maybe a dependencyManagement section somewhere needs to be updated?

Reply:
Found the upstream reference in the main pom.xml and fixed it there.